README.md

```diff
@@ -139,6 +139,24 @@ updateJob = batchSQLClient.update(job_id, NEW_QUERY)
 cancelJob = batchSQLClient.cancel(job_id)
 ```
 
+**COPY queries**
+
+COPY queries allow you to use the [PostgreSQL COPY command](https://www.postgresql.org/docs/10/static/sql-copy.html) for efficient streaming of data to and from CARTO.
+
+```python
+from carto.sql import CopySQLClient
+
+copyClient = CopySQLClient(auth_client)
+
+# COPY FROM example
+query = 'COPY copy_example (the_geom, name, age) FROM stdin WITH (FORMAT csv, HEADER true)'
+result = copyClient.copyfrom_file_path(query, 'copy_from.csv')
```

Comment:
wouldn't it be more pythonic to take a file object rather than an actual file name? Not that a convenience method for a filename isn't good, but at core, shouldn't the API be exposing a [file](https://docs.python.org/2.4/lib/bltin-file-objects.html)?

Reply:
As part of the public API, it does expose the `file` and even accepts a lower level "iterable object" :slightly_smiling_face: 

https://github.com/CartoDB/carto-python/blob/655e08308002372f4817863001b138309cf01586/carto/sql.py#L287-L348

and there are nice examples of usage in the tests:

https://github.com/CartoDB/carto-python/blob/655e08308002372f4817863001b138309cf01586/tests/test_sql_copy.py

I'll add a "more pythonic" `file` object example to the README.